src/cljs/athens/core.cljs

```diff
@@ -29,13 +30,20 @@
                 (getElement "app")))
 
 
+;(let [agent (.. js/navigator -userAgent)
+;      idx (clojure.string/index-of agent "Electron/")]
+;  (when (< -1 idx)
+;    (require '[athens.electron])))
```

Comment:
This is how you can determine, from the frontend, whether the chromium environment is Electron or browser based. Not sure if it works in practice, where you selectively require.